src/main.rs

```diff
@@ -145,6 +182,230 @@ async fn serve_req(
             .body(Body::from(triagebot::zulip::respond(&ctx, req).await))
             .unwrap());
     }
+
+    if req.uri.path() == "/review-settings" {
```

Comment:
`/review-settings` is a new endpoint of the Triagebot that will allow access to the backoffice to set the pull request assignment preferences.

Here there is quite some code to authenticate the user. could probably be split and refactored a bit for better legibility